frontend/pages/settings/index.tsx

```diff
@@ -27,9 +27,10 @@ export default function Settings() {
   if (error) return error
 
   return (
-    <Wrapper fullWidth title='Settings' page='settings'>
+    <>
+      <Title title='Settings' />
       {isCurrentUserLoading && <Loading />}
       <PageWithTabs title='Settings' tabs={tabs} />
```

Comment:
I think we need a `<Container maxWidth='xl'>` wrapping this

Reply:
Container's give a padding on both height and width, whereas we want any page with the tabbed view _not_ to have padding on the width.

Reply:
You're entirely right, I thought I'd removed all the comments on tab views. My mistake!